Resources/Markdown.md

```diff
@@ -0,0 +1,54 @@
+# Markdown
+
+Markdown was designed to be readable as-is, without looking like it has been marked up with tags or formatting instructions (like HTML). It is a markup language commonly used for writing simply formatted documents without needing to use a GUI to format text, such as using **bold** and *italics*. Markdown allows you to write using an easy-to-read and easy-to-write plain text format.
+
+Documentation on programming projects is often lacking. Markdown is an important language for developers to master because it provides a 
+means to discuss a project and features using a language that all web 
+browsers and integrated development environments (IDEs) can understand, and its use across applications (GitHub, Slack, Notion, etc.).. Far 
+easier than hypertext markup language (HTML) to learn, basic Markdown can be 
+learned in a few minutes. 
+
+## Links / Resources
+### Cheat Sheet
+***Description***: 
+- Provides a quick overview of all Markdown syntax elements.
+
+***Examples Provided***: 
+* Basic Syntax: Heading, Bold, Italic, Blockquote, Ordered List, 
+    Unordered List, Code, Horizontal Rule, Link, Image
+* Extended Syntax: Table, Fenced Code Block, Footnote, Heading ID, 
+    Definition List, Strikethrough, Task List, Emoji, Highlight, 
+    Subscript, Superscript)
+    
+***Downloads Provided***: 
+- Cheat sheet as a Markdown file
+
+***Resource***: 
+- [Markdown Cheat Sheet](https://www.markdownguide.org/cheat-sheet/)  
+
+### Communicate using Markdown
+***Description***:
+
+-  Making clean and organized information is important in software collaboration. Using markdown allows developers to communicate information more effectively as well as helps guide and instruct on how to use an application. This allows for better pull requests, issue handeling, and readability. The provided resource will guide you through using markdown to update a plain text file.
+
+***Examples***:
+- Creating headings
+- [] Creating check boxes
```

Comment:
Is there a reason these brackets are here?

Reply:
is it not rendering the checkbox? I see vs code was having issue rendering the checkbox, Its not necessary per say, we can remove

Reply:
I removed the brackets, its under a new pull request as im not over of the forked repo

Reply:
I'll take a look!  I think in markdown there need's to be a space between the brackets for it to render.  I am happy with it just being a bullet point.

I'll take a look the other PR!